src/utilities/aggregateLogs.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+/**
+ * Aggregates sensor logs into groups of logs by dates.
+ *
+ * @param   {Realm.results}  sensorLogs          A collection of sensorLog objects to aggregate.
+ * @param   {boolean}        isMax               If true, aggregate by maximum temperature, else
+ *                                               by minimum temperature.
+ * @param   {number}         numberOfDataPoints  Number of aggregated data points to return.
+ * @param   {Date}           startDate           Start date of temperature range to aggregate
+ *                                               over.
+ * @param   {Date}           endDate             End date of temperature range to aggregate over.
+ * @return  {Object[]}                           Aggregated sensor logs, array of objects in form
+ *                                              [{timestamp}, {temperature},...].
+ */
+export const aggregateLogs = ({
+  sensorLogs,
+  numberOfIntervals,
+  isMax = true,
+  startDate = new Date(),
```

Comment:
If no `startDate` is passed, the `startDate` will always be today, rather than the first `timestamp` of `sensorLogs` unless your `sensorLogs` are from the future.

Reply:
I thought it would initialise as epoch time (which still would have been a problem, so not defence!) Fixed by 6f9f29dd.